### ISSUE TITLE
Issue in cross checking the flows/subflows in the directory

### ISSUE THREAD

**Issue by aarjavshah** [EXXETA/sonar-msgflow-plugin#16]
Encountered the new issue. While running the rule for the Unused flows, false positive is thrown when the flows/subflow referenced are present inside a folder. For Example, I have a folder structure something like this.
```
Application1
    -->Msgflow1.msgflow
    -->dir
          -->dir1
               -->sbflow1.suflow
               -->sbflow2.subflow


```
when those two subflows are used in a Msgflow1.msgflow they are referenced using something like this.
dir_dir1_sbflow1.subflow and so on. However, in the current implementation when load the list of all the flows we get it with dir/dir1/subflow1.subflow 

So, while cross checking the list with the used subflows, The ones present in the flow does not match to the corresponding flow from the list of flows.

I've identified the fix for this and will push the code for the same.

**Comment by aarjavshah** [comment]
The fix is merged to the EXXETA branch.
